config/samples/infra_v1alpha1_cluster.yaml

```diff
@@ -1,35 +1,32 @@
 apiVersion: infra.nephio.org/v1alpha1
 kind: Cluster
 metadata:
-  name: nephio-regional-01
+  name: large-autoscaling-site
```

Comment:
Can we keep the original names and rename the clusterScaleProfiles instead?

Reply:
done